src/test/java/gov/usgs/wma/waterdata/TransformDailyValueIT.java

```diff
@@ -27,6 +23,8 @@
 import com.github.springtestdbunit.annotation.ExpectedDatabase;
 import com.github.springtestdbunit.assertion.DatabaseAssertionMode;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I actually prefer importing each thing individually rather than using *.  Same for the other test.

Reply:
Will fix.  This is intellij being too helpful...

Reply:
fixed on latest commit